CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# generated by giantswarm/github actions - changes will be overwritten
+* @giantswarm/team-atlas
```

Comment:
![image](https://github.com/giantswarm/grafana-agent-app/assets/12008875/d98caabe-1123-41d7-b769-6b9225b68800)
The team is not publicly visible maybe? Is it expected?